docs/sources/operators-guide/architecture/components/ingester.md

```diff
@@ -63,6 +64,14 @@ If an ingester fails, a subsequent process restart replays the WAL and recovers
 Contrary to the sole replication, and given that the persistent disk data is not lost, in the event of the failure of multiple ingesters, each ingester recovers the in-memory series samples from WAL after a subsequent restart.
 Replication is still recommended in order to gracefully handle a single ingester failure.
 
+### Write-behind log
+
+The write-behind log (WBL) is very much like the WAL but it only writes incoming out-of-order samples to a persistent disk until the series are uploaded to long-term storage.
+
+The reason why there is a different log for this is that it is not possible to know if a sample is out-of-order until we try to append it. So we first need to attempt it, detect that it is out-of-order, append it anyway if out-of-order is enabled and then write it to the log.
```

Comment:
```suggestion
There is a different log for this because it is not possible to know if a sample is out-of-order until you try to append to it. First you need to attempt to append to it, detect that it is out-of-order, append to it anyway if out-of-order is enabled, and then write the sample to the log.
```
Please vet this for technical accuracy.

Reply:
We --> you, so you are addressing your reader directly.

Reply:
Its appending to the in memory series chunks that is why I said `append it`. `Do you not know if its out-of-order unless you try to append it` I'm omitting the in memories series chunk because I think its too technical concept.

Reply:
Fixed with this 5115dcf4f4bcd5db35cc035bc00d471d812d234f